packages/@okta/vuepress-site/docs/release-notes/2024-okta-identity-engine/index.md

```diff
@@ -6,6 +6,78 @@ title: Okta Identity Engine API Products release notes 2024
+
+#### New mappings property for Policy API is EA in Preview
+
+A new `mappings` property is available for the `links` object in  `GET /api/v1/policies/${policyId}` and `GET /api/v1/policies?type=${type}` responses. This property displays links to policy mappings. See [Policy API](/docs/reference/api/policy/#links-object). <!-- OKTA-637310 -->
+
+#### My Account Authenticators API is GA in Production
+
+The MyAccounts Authenticators API (`/idp/myaccount/authenticators/`) enables you to list enrolled and un-enrolled authenticator information. You can also access details of specific authenticators and enrollments. <!-- OKTA-670703 -->
+
+#### AAL values for Login.gov IdP
+
+The [Login.gov IdP configuration](/docs/guides/add-logingov-idp/main/#create-an-identity-provider-in-okta) has been updated to include all allowed AAL values. <!-- OKTA-673125 -->
+
+#### Granular API policy authenticator controls is GA in Preview
+
+The Authentication Policy API now includes three new `constraints` object parameters that provide precise control over what specific authenticators and methods are displayed to end users. Previously, some authenticators were mapped to the same authenticator `types` and `methods`. The parameters `authenticationMethods` and `excludeAuthenticationMethods` now identify (or exclude) the exact authenticator for both `knowledge` and `possession` constraints. The `required` parameter indicates whether the `knowledge` or `possession` constraints are required by the assurance. See the [Policy API](/docs/reference/api/policy/#constraints). <!--OKTA-676880 ASSURANCE_GRANULAR_AUTHENTICATOR_CONSTRAINTS -->
+
+#### Externally signed org AS access tokens
+
+Access tokens returned from the org authorization server are now signed using the externally published signing key. These access tokens must still be treated as opaque strings and not be validated or consumed by any application other than Okta. <!-- OKTA-694170 -->
+
+#### Granular controls for authentication policies
+
+You can now disallow or allow individual authentication methods for an authentication policy. This gives admins more  granular control over access to apps. <!-- OKTA-699297 -->
+
+#### Support case management for admins is GA in Preview
+
+Super admins can now assign the View, create, and manage Okta support cases permission and Support Cases resource to a custom admin role. This allows delegated admins to manage the support cases that they’ve opened. See [About role permissions](https://help.okta.com/okta_help.htm?type=oie&id=csh-about-role-permissions). <!-- OKTA-700229 -->
```

Comment:
This needs to be set apart so that it's easier to read.

```suggestion
Super admins can now assign the **View, create, and manage Okta support cases** permission and Support Cases resource to a custom admin role. This allows delegated admins to manage the support cases that they’ve opened. See [About role permissions](https://help.okta.com/okta_help.htm?type=oie&id=csh-about-role-permissions). <!-- OKTA-700229 -->
```